File: algorithms/1184. Distance Between Bus Stops/solution.cc
class Solution {
public:
    int distanceBetweenBusStops(vector<int>& distance, int start, int destination) {
        if (start > destination) swap(start, destination);
        int clockwise = accumulate(distance.begin() + start, distance.begin() + destination, 0);
        int counterclockwise = accumulate(distance.begin() + destination, distance.end(), 0) + accumulate(distance.begin(), distance.begin() + start, 0) ;
        return min(clockwise, counterclockwise);
    }
};

File: algorithms/1184. Distance Between Bus Stops/solution.py
class Solution:
    def distanceBetweenBusStops(self, distance: List[int], start: int, destination: int) -> int:
        l, r = min(start, destination), max(start, destination)
        return min(sum(distance[l:r]), sum(distance[:l]) + sum(distance[r:]))
